app-ios/Sources/SearchFeature/SearchedSessionTimeView.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+
+struct SearchedSessionTimeView: View {
```

Comment:
How about create common component and use in Search and Timetable?
A component Inside `TimetableListView` has almost all the same as this.

Reply:
I agree !

22b4bc6
2d0fb83